third_party/maya/lib/usdMaya/usdExport.cpp

```diff
@@ -119,21 +119,15 @@ try
     JobExportArgs jobArgs;
 
     if (argData.isFlagSet("mergeTransformAndShape")) {
-        bool tmpBool = true;
-        argData.getFlagArgument("mergeTransformAndShape", 0, tmpBool);
-        jobArgs.mergeTransformAndShape = tmpBool;
+        argData.getFlagArgument("mergeTransformAndShape", 0, jobArgs.mergeTransformAndShape);
     }
 
     if (argData.isFlagSet("exportRefsAsInstanceable")) {
-        bool tmpBool = false;
-        argData.getFlagArgument("exportRefsAsInstanceable", 0, tmpBool);
-        jobArgs.exportRefsAsInstanceable = tmpBool;
+        argData.getFlagArgument("exportRefsAsInstanceable", 0, jobArgs.mergeTransformAndShape);
```

Comment:
Yes... yes I did, my apologies!

I submitted a new PR with an ammended commit - if you'd prefer a new patch commit, let me know, and I can do that instead.